assets/queries/cloudFormation/amplify_app_access_token_rule/test/negative.yaml

```diff
@@ -0,0 +1,59 @@
+Resources:
+     NewAmpApp-2:
+        Type: AWS::Amplify::App
+        Properties:
+          AccessToken: !Sub '{{resolve:secretsmanager:${MyAmpAppSecretManagerRotater}::password}}'
+          BuildSpec: String
+          CustomHeaders: String
+          Description: String
+          EnableBranchAutoDeletion: true
+          IAMServiceRole: String
+          Name: NewAmpApp
+          OauthToken: String
+          Repository: String
+     MyAmpAppSecretManagerRotater:
+        Type: AWS::SecretsManager::Secret
+        Properties:
+          Description: 'This is my amp app instance secret'
+          GenerateSecretString:
+            SecretStringTemplate: '{"username": "admin"}'
+            GenerateStringKey: 'password'
+            PasswordLength: 16
+            ExcludeCharacters: '"@/\'
+---
+Parameters:
+  ParentAccessToken:
+    Description: 'Access Token'
+    Type: String
+Resources:
+  NewAmpApp-1:
```

Comment:
Remove `-`
Linter message: `Resource NewAmpApp-1 has an invalid name.  Name has to be alphanumeric.`